cpp/src/DataStorm/TopicI.cpp

```diff
@@ -57,6 +39,54 @@ namespace
         bool match(const shared_ptr<Filterable>&) const final { return true; }
     };
     const auto alwaysMatchFilter = make_shared<AlwaysMatchFilter>();
+
+    DataStorm::ClearHistoryPolicy parseClearHistory(const std::string& value)
+    {
+        if (value == "OnAdd")
+        {
+            return DataStorm::ClearHistoryPolicy::OnAdd;
+        }
+        else if (value == "OnRemove")
+        {
+            return DataStorm::ClearHistoryPolicy::OnRemove;
+        }
+        else if (value == "OnAll")
+        {
+            return DataStorm::ClearHistoryPolicy::OnAll;
+        }
+        else if (value == "OnAllExceptPartialUpdate")
+        {
+            return DataStorm::ClearHistoryPolicy::OnAllExceptPartialUpdate;
+        }
+        else if (value == "Never")
+        {
+            return DataStorm::ClearHistoryPolicy::Never;
+        }
+        else
+        {
+            throw Ice::ParseException(__FILE__, __LINE__, "Invalid clear history policy: " + value);
+        }
+    }
+
+    DataStorm::DiscardPolicy parseDiscardPolicy(const std::string& value)
+    {
+        if (value == "Never")
+        {
+            return DataStorm::DiscardPolicy::None;
+        }
```

Comment:
We have this discrepancy, documented value is "Never", but the enum value is None.

Reply:
I assume `Never` is the "right" choice here.

Reply:
DiscardPolicy is part of the public API. User code can still set it manually using `DiscardPolicy::None`.

Should we rename `DiscardPolicy::None` to `DiscardPolicy::Never`?

Reply:
I think it's preferable to just have one value.